src/dbg/commands/cmd-gui.cpp

```diff
@@ -46,6 +48,16 @@ bool cbDebugDump(int argc, char* argv[])
         }
         GuiDumpAtN(addr, int(index));
     }
+    else if(strcmp(activeView.title, "Dump 1") == 0)

```

Comment:
This would probably be better to do as a regex to support additional dumps, but otherwise looks good!